<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>497</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TCGP Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="crawlerTab">
         <attribute name="title">
          <string>Crawler</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="exorpSelection">
            <item>
             <widget class="QRadioButton" name="expRadioBtn">
              <property name="text">
               <string>Expansion</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="packRadioBtn">
              <property name="text">
               <string>Pack Key</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="crawlerGB">
            <property name="title">
             <string>Pack Key</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QGroupBox" name="expCB">
               <property name="title">
                <string>Code</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QComboBox" name="expComboB"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="packKeyGB">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="title">
                <string>Pack key</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QComboBox" name="packKeyComboB"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="startCrawlingBtn">
            <property name="text">
             <string>Start Crawling</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="crawlerProgressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="imageRenamerTab">
         <attribute name="title">
          <string>Image Renamer</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Folder Selection</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="folderLineEdit">
                 <property name="locale">
                  <locale language="English" country="UnitedStates"/>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string>No folders selected</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="browseFolderBtnInTab2">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                 <property name="icon">
                  <iconset theme="folder-open">
                   <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <property name="autoRepeat">
                  <bool>false</bool>
                 </property>
                 <property name="autoExclusive">
                  <bool>false</bool>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                 <property name="default">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearFoldersBtnInTab2">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                 <property name="icon">
                  <iconset theme="edit-clear">
                   <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Selected Folders:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="folderListWidget">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="countLabel">
                 <property name="text">
                  <string>Total folders: 0</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="removeSelectedBtnInTab2">
                 <property name="text">
                  <string>Remove Selected</string>
                 </property>
                 <property name="icon">
                  <iconset theme="list-remove">
                   <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Excel File</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLineEdit" name="fileLineEdit">
               <property name="locale">
                <locale language="English" country="UnitedStates"/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>No folders selected</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseFileBtnInTab2">
               <property name="text">
                <string>Browse...</string>
               </property>
               <property name="icon">
                <iconset theme="folder-open">
                 <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearFileBtnInTab2">
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="icon">
                <iconset theme="edit-clear">
                 <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QCheckBox" name="dryRunCB">
              <property name="text">
               <string>Dry Run</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="startRenameBtn">
              <property name="text">
               <string>Start Renamer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="renamerProgressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="genCardJsonTab">
         <attribute name="title">
          <string>Gen Card Json</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Expansion</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QComboBox" name="expansionComboBox"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Folder Selection</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLineEdit" name="folderLineEditInTab3">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string>No folders selected</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="browseFolderBtnInTab3">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                 <property name="icon">
                  <iconset theme="folder-open">
                   <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearBtnInTab3">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                 <property name="icon">
                  <iconset theme="edit-clear">
                   <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Excel Files Selection</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLineEdit" name="filesLineEditInTab3">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string>No folders selected</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="browseExcelBtnInTab3">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                 <property name="icon">
                  <iconset theme="folder-open">
                   <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearExcelBtnInTab3">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                 <property name="icon">
                  <iconset theme="edit-clear">
                   <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Selected Files:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="excelListWidget"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="countExcelLabel">
                 <property name="text">
                  <string>Total Excel Files: 0</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="removeSelectedBtnInTab3">
                 <property name="text">
                  <string>Remove Selected</string>
                 </property>
                 <property name="icon">
                  <iconset theme="list-remove">
                   <normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</normaloff>C:/Users/sam99/AppData/Local/Temp/Rar$DRa61588.31932.rartemp</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="startGenBtn">
              <property name="text">
               <string>Start Generation</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="genJsonProgressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>expRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>packKeyGB</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>packRadioBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>packKeyGB</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
